Patch by Robert Scheck <robert@fedoraproject.org> for eggdrop >= 1.6.17, which
replaces the hardcoded language directory path to avoid the use of EGG_LANGDIR
environment variable per default (#194481 #c6-c9).

--- eggdrop1.6.17/src/eggdrop.h			2004-07-25 13:17:34.000000000 +0200
+++ eggdrop1.6.17/src/eggdrop.h.langdir		2006-06-18 01:19:23.000000000 +0200
@@ -75,7 +75,7 @@
 
 
 /* Language stuff */
-#define LANGDIR  "./language" /* language file directory                   */
+#define LANGDIR  "/usr/share/eggdrop/language"  /* language file directory */
 #define BASELANG "english"    /* language which always gets loaded before
                                  all other languages. You do not want to
                                  change this.                              */
--- eggdrop1.6.17/doc/man1/eggdrop.1		2004-08-09 05:19:01.000000000 +0200
+++ eggdrop1.6.17/doc/man1/eggdrop.1.langdir	2006-06-18 01:19:23.000000000 +0200
@@ -115,7 +115,7 @@
 .TP
 .B EGG_LANGDIR
 Specifies the directory where all your language files are stored.
-The default is ./language.
+The default is /usr/share/eggdrop/language.
 .SH "SEE ALSO"
 tclsh(1),
 irc(1),
